manifests/overlays/additional/kustomization.yaml

```diff
@@ -313,19 +313,19 @@ replacements:
           kind: ImageStream
           name: runtime-datascience-cpu-py312-ubi9
           version: v1
-  # - source:
-  #     fieldPath: data.odh-pipeline-runtime-pytorch-llmcompressor-cuda-py312-ubi9-n
-  #     kind: ConfigMap
-  #     name: notebook-image-params
-  #     version: v1
-  #   targets:
-  #     - fieldPaths:
-  #         - spec.tags.0.from.name
-  #       select:
-  #         group: image.openshift.io
-  #         kind: ImageStream
-  #         name: runtime-pytorch-llmcompressor-cuda-py312-ubi9
-  #         version: v1
+  - source:
+      fieldPath: data.odh-pipeline-runtime-pytorch-llmcompressor-cuda-py312-ubi9-n
+      kind: ConfigMap
+      name: notebook-image-params
+      version: v1
+    targets:
+      - fieldPaths:
+          - spec.tags.0.from.name
+        select:
+          group: image.openshift.io
+          kind: ImageStream
+          name: runtime-pytorch-llmcompressor-cuda-py312-ubi9
+          version: v1
```

Comment:
why are there only three `source:` blocks like this and not 4? isn't one missing?

@coderabbitai think on it too please

Reply:
@coderabbitai what about `runtime-pytorch-llmcompressor-cuda-py312-ubi9` having its commit hash replacement? that's what's missing

Reply:
runtime images are intentionally without commit hash tracking